docs/using.md

```diff
@@ -171,41 +489,294 @@ is passed in `infrastructure_provisioning: bar` and the third `arbitrary_key: ba
+created until a workload actor requests them.
+
+The `Default` pool is what actors connect to unless their .cpp class determines otherwise. Some actors (the Loader, CrudActor, and RunCommand actors)
+have a `ClientName` field for specifying which pool to request.
+
+Genny's preprocessor operates on this configuration to make using it easier. The `-u` CLI option can be used to set the default URI. During preprocessing,
+any pool that does not have the `URI` key set will be given this value. The default value of the default URI is `"mongodb://localhost:27017"`. Since URI
+is typically not known until runtime, this means that most workloads should have a configuration more like the following:
+
+    Clients:
+      Default:
+        QueryOptions:
+          maxPoolSize: 500
+      Update:
+        QueryOptions:
+          maxPoolSize: 500
+
+
+<a id="orga591018"></a>
+
+### Multiple Connection Strings
+
+If multiple different connection strings are needed, such as when testing a multitenant system, we can use an override file. For the above configuration,
+we can create the following override:
+
+    Clients:
+      Default:
+        URI: "mongodb://localhost:27017"
+      Update:
+        URI: "mongodb://localhost:27018"
+
+Notice the different ports. This can be used at runtime as:
+
+    ./run-genny workload example.yml -o override.yml
+
+This will apply the override onto the workload, creating the following result:
+
+    Clients:
+      Default:
+        QueryOptions:
+          maxPoolSize: 500
+        URI: "mongodb://localhost:27017"
+      Update:
+        QueryOptions:
+          maxPoolSize: 500
+        URI: "mongodb://localhost:27018"
+
+Genny's `evaluate` subcommand can always be used to see the result of complex configurations.
+
+
+<a id="org65830c2"></a>
+
+### Default
+
+Since actors generally need a connection and not all workloads need a complicated connection or multiple pools,
+simply not setting any connection pools will cause Genny to default to the following:
+
+    Clients:
+      Default:
+        QueryOptions:
+          maxPoolSize: 100
+        URI: "mongodb://localhost:27017"
+
+For more information, see [Defaults and Overrides](#org22b7a0f).
+
+
+<a id="org7e6c6bd"></a>
+
+## Creating an Actor
+
+Creating new actors is a common and encouraged workflow in Genny. To create one, run the following:
+
+    ./run-genny create-new-actor
+
+This will create new actor .cpp and .h files, an example workload yaml, as well as actor integration tests, all with inline comments guiding you through the actor creation process. You might want to take a look at [Developing Genny](./developing.md) and the [Contribution Guidelines](../CONTRIBUTING.md).
+
+It is encouraged to make new actors as general as possible, for reuse among workloads. No need to loop TIPS into a PR when developing an actor, unless you'd just prefer a second look. Actor authors own their actors.
```

Comment:
It's also entirely possible and good for Actors to be written for very intricate use-cases. In many cases 500 lines of CrudActor would be much clearer in a custom cpp. I dont' think general/reusable is any bar for entry for new Actors. "If your configuration wants to use logic, ifs, or anything beyond simple or existing commands in a loop, then consider writing your own Actor. It doesn't need to be super general or even super well-tested or refactored. Genny is open to submissions and you own whatever Actor you write." Something like that idk.